R/copy-model-from.R

```diff
@@ -5,11 +5,11 @@
 #' Create new model by copying existing model
 #'
 #' Create new model by copying existing model. Useful for iterating during model development.
-#' Also fills `based_on` field by default, for constructing model ancestry. See \href{../articles/using-based-on.html}{"Using based_on field" vignette} for details.
+#' Also fills `based_on` field by default, for constructing model ancestry. See ["Using based_on field" vignette](../articles/using-based-on.html) for details.
 #' @param .parent_mod Model to copy from
 #' @param .new_model Path to write new model files to WITHOUT FILE EXTENSION. Function will create both `{.new_model}.yaml` and a new model file based on this path.
-#' @param .description Description of new model run. This will be stored in the yaml (to be used later in `create_run_log()`).
-#' @param .based_on_additional Character scaler or vector of paths to other models that this model was "based on." These are used to reconstuct model developement and ancestry. \strong{Paths must be relative to `.new_model` path.} Note that the `.parent_model` will automatically be added to the `based_on` field, so no need to include that here.
+#' @param .description Description of new model run. This will be stored in the yaml (to be used later in `run_log()`).
+#' @param .based_on_additional Character scaler or vector of paths to other models that this model was "based on." These are used to reconstuct model developement and ancestry. **Paths must be relative to `.new_model` path.** Note that the `.parent_model` will automatically be added to the `based_on` field, so no need to include that here.
 #' @param .add_tags A character scaler or vector with any new tags to be added to `{.new_model}.yaml`
 #' @param .inherit_tags Boolean for whether to inherit any tags from `.parent_mod`
```

Comment:
The Boolean type is more commonly referred to by its R type name, `logical`. That said, it may be more helpful to phrase the description for logical arguments in terms of what will happen in the relevant cases, e.g., 

```
@param When `TRUE` (the default), the new model will inherit tags from `.parent_mod`. Set to `FALSE` to avoid this.
```

Reply:
Changed all these. It felt weird at first to not declare the type, but it does read nicer.